monitoring/prober/test_isa_validation.py

```diff
@@ -21,24 +21,7 @@ def test_isa_huge_area(session, isa1_uuid):
           'extents': {
               'spatial_volume': {
                   'footprint': {
-                      'vertices': [
-                          {
-                              'lat': 130,
```

Comment:
Something is fishy here...5034048745.815665km² (the "expected" error message below) is substantially larger than the surface area of the earth so this seems like an invalid error message (true error message should be something about invalid latitudes with current input).  Let's make sure to add an issue to figure out what was going on, add a test to reproduce, and fix the problem.